src/components/data/data-utils.js

```diff
@@ -346,8 +346,9 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
     }
   })
 
-  const match = /(?:membership|entity):([^\s]+)_id/.exec(fileContents)
-  const isInvalid = dataProvider.tsvFeatures.isInvalid({ fileImportModeMatches: isFileImportCurrMode === isFileImportLastUsedMode, match: !match, filePresent: file, sysNamePresent: fileContents.split('\n')[0].match(/sys_name/) }) //only look at the first line
+  // const match = isGoogleWorkspace ? /(?:membership|entity):([^\s]+)_id/.exec(fileContents) : !isGoogleWorkspace;
```

Comment:
nit: delete the commented-out line

Reply:
whoops forgot to push that fix -- updated!